app/models/office_recipient.rb

```diff
@@ -19,7 +19,17 @@ def office
     if office_location.present?
       self.class.offices[office_location]
     else
-      self.class.find_by(zip: residential_address_zip)
+      self.class.find_by(zip: residential_or_mailing_zip)
+    end
+  end
+
+  def residential_or_mailing_zip
+    if benefit_application.respond_to?(:residential_zip)
```

Comment:
I think we don't use `residential_zip` anymore—seems to be legacy that no longer is meaningfully in the code base. can we remove entirely?